docs/hbase.txt

```diff
@@ -173,8 +165,7 @@ hadoop version # Should print 1.0.3
 hadoop fs -ls /
 ----
 
-Tips and Tricks for Managing an HBase Cluster
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+=== Tips and Tricks for Managing an HBase Cluster
```

Comment:
EMR+HBase added to issue for follow-up. Bigtable directions already added in separate PR.